src/analysis/retail/priest/holy/modules/talents/BottomRow/AnsweredPrayers.tsx

```diff
@@ -0,0 +1,48 @@
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import TALENTS from 'common/TALENTS/priest';
+import SPELLS from 'common/SPELLS';
+import Events, { HealEvent } from 'parser/core/Events';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
+import { SpellLink } from 'interface';
+
+//Example log: /report/cCKp6qfAM8FZgxHL/2-Heroic+Broodkeeper+Diurna+-+Wipe+2+(1:34)/Mayceia/standard/statistics
+class AnsweredPrayers extends Analyzer {
+  prayerHeals = 0;
+  healsPerApotheosis = 0;
+  healsPerTrigger = [0, 100, 50];
```

Comment:
I should have been more clear about moving this value - it should still be a `const`, just in this file instead of another one. The style we've been using in WoWA is
```ts
const HEALS_PER_TRIGGER_BY_RANK = [0, 100, 50];

class AnsweredPrayers extends Analyzer {
...
```